create_masks.py

```diff
@@ -0,0 +1,93 @@
+import os
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "deepgis_moon.agdss.settings")
+django.setup()
+
+from wand.color import Color as WandColor
```

Comment:
Let's add wand and other dependencies to a new requirements.txt file in the repo, so that users can pip install easily.
This can be a separate commit.